src/lib/classifyquote.tsx

```diff
@@ -39,7 +45,7 @@ melanie
 [5:23 PM]
 I just got home, so I'm gonna read them now!
 */
-const slackHeaderRegex = /^(.+?[^\s])( |$\n?)\[\d{1,2}:\d{2} (A|P)M\](\s+)?/;
+const slackVerboseHeaderRegex = /^(.+?[^\s])( |$(\r\n|\n)?)\[\d{1,2}:\d{2} (A|P)M\](\s+)?/;
```

Comment:
wrt the regexes:  I'm just trying to disambiguate `\n` or `\r\n`, if we used `[\r\n]{1,2}`, it could capture sequences like `\n\n` or `\r\r`. I'm honestly not sure how to group the pipe operator without using parenthesis.

I didn't realize js regex supported non-capturing groups, so we can use those here.